lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -2255,25 +2255,52 @@ def jsonrpc_configure_ui(self, p):
         return d
 
     @AuthJSONRPCServer.auth_required
+    def jsonrpc_open(self, p):
```

Comment:
write this using inlineCallbacks:

@defer.inlineCallbacks
def jsonrpc_open(self, p):
    """<docstring>"""
    if 'sd_hash' not in p:
        raise ValueError('sd_hash is required')
    lbry_file = yield self._get_lbry_file(FileID.SD_HASH, p['sd_hash'])
    if not lbry_file:
        raise Exception('Unable to find file for {}'.format(p['sd_hash']))
    try:
        file_utils.start(lbry_file['download_path'])
    except IOError:
        pass
    defer.returnValue(True)
```